.changeset/nice-grapes-show.md

```diff
@@ -0,0 +1,45 @@
+---
+"@fluidframework/merge-tree": minor
+"@fluidframework/sequence": minor
+---
+---
+"section": legacy
+---
+
+Remove Merge-Tree and SharedString ISegment Deprecations
```

Comment:
Nit (since we're removing the actual APIs and not removing their deprecation status):
```suggestion
Remove deprecated Merge-Tree and SharedString ISegment members
```